_includes/docs/latest/setup-cloudsql.md

```diff
@@ -7,6 +7,8 @@ To set up your Cloud SQL instance for Forseti, follow the steps below:
     follow the steps below to create a new instance:
     1. Select a **MySQL** database engine.
     1. Select a **Second Generation** instance type.
+    1. On the **Choose use case** page, select the configuration type you want based upon
```

Comment:
Hmmm, I do not see this `Choose use case` page in the cloud console.  

Instead, I see:
1: Choose a MySQL instance type: MySQL, 2nd generation
2: Create a MySQL Second Generation instance